docs/detections/alerts-visualizations.asciidoc

```diff
@@ -15,7 +15,9 @@ For example, you can group first by rule name (`Group by: kibana.alert.rule.name
 
 NOTE: Some view types don't have the *Group by top* option. You can also leave *Group by top* blank to group by only the primary field in *Group by*.
 
-To reset a view to default settings, click the settings menu (image:images/three-dot-icon.png[Settings menu icon,18,18]) in the upper-right and select *Reset group by fields* (the settings menu appears when you hover over the area).
+To reset a view to default settings, hover over it and click the options menu (image:images/three-dot-icon.png[Settings menu icon,18,18]) that appears, then select *Reset group by fields*. 
```

Comment:
See below [question](https://github.com/elastic/security-docs/pull/3298/files#diff-104ec26f4b74ea5b31e353ff2bda7804f7e927ee2346bba1a8f8c1ab4edbcb80R44) about menus. More of a question.